regression/goto-instrument/remove-function-body1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--remove-function-body foo --remove-function-body bar
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
I have added patterns that must _not_ be found in test.desc, so I think this is addressed.